adminlogin.aspx.cs

```diff
@@ -27,7 +27,7 @@ protected void Button1_Click(object sender, EventArgs e)
                 {
                     con.Open();
                 }
-                SqlCommand cmd = new SqlCommand("select * from admin_login_tbl where username='" + TextBox1.Text.Trim() + "' AND password='" + TextBox2.Text.Trim() + "'", con);
+                SqlCommand cmd = new SqlCommand("select * from admin_login_tbl where username='" + txtAdminId.Text.Trim() + "' AND password='" + txtPassword.Text.Trim() + "'", con);
```

Comment:
### Split into multiple statements. 

Good that you corrected the names but it's still in poor taste. 

Please split this into multiple statements. 

```suggestion
string selectQuery = $@"SELECT * FROM admin_login_tbl WHERE username = @username AND password = @password";

SqlCommand cmd = new SqlCommand(selectQuery, con);
cmd.Parameters.Clear();
cmd.Parameters.AddWithValue("@username", txtAdminId.Text.Trim());
cmd.Parameters.AddWithValue("@password", txtPassword.Text.Trim());

```